plugins.py

```diff
@@ -0,0 +1,21 @@
+from courseware.tabs import EnrolledTab
+from django.conf import settings
+from django.utils.translation import ugettext_noop
+
+
+class NodeBBTab(EnrolledTab):
+    """
+    NodeBBTab for Courses it will contain the embeded view of NodeBB Forum
+    """
+
+    name = 'openedx_nodebb_discussion'
+    type = 'openedx_nodebb_discussion'
```

Comment:
`type` is a reserved keyword. Change the variable name.

Reply:
It's also reserved word for edX Enrolled Tab class and its necessary here.